setup.py

```diff
@@ -301,9 +301,7 @@ def write_version(filename: str = str(AIRFLOW_SOURCES_ROOT / "airflow" / "git_ve
 ]
 leveldb = ["plyvel"]
 otel = ["opentelemetry-api==1.15.0", "opentelemetry-exporter-otlp", "opentelemetry-exporter-prometheus"]
-pandas = [
-    "pandas>=0.17.1",
-]
+pandas = ["pandas>=0.17.1", "pyarrow>=9.0.0"]
```

Comment:
pyarrow is too heavy though and previously had compiling issues on buster, maybe not anymore

Reply:
Buster is EOL? https://wiki.debian.org/DebianReleases